Cartfile.resolved

```diff
@@ -1,3 +0,0 @@
-github "JanGorman/Hippolyte" "0.6.0"
-github "httpswift/swifter" "1.4.6"
-github "hyperwallet/hyperwallet-ios-sdk" "5771c52cd1303a5c68f0f2589df7c0d52fb19e78"
```

Comment:
why removed this?

Reply:
I think we can do the same in Carthage file (I mean fix packages versions)